src/main.rb

```diff
@@ -0,0 +1,163 @@
+require_relative './person'
+require_relative './student'
+require_relative './rental'
+require_relative './book'
+require_relative './teacher'
+require_relative './classroom'
+
+# Initialize the App class
+class App
+  def initialize
+    @books = []
+    @rentals = []
+    @people = []
+  end
+
+  def run
+    puts 'Welcome to School Library App!'
+
+    loop do
+      actions
+
+      option = gets.chomp
+
+      break if option == '7'
+
+      handle_action option
+    end
+
+    puts 'Thank you for using this app!'
+  end
+
+  private
+
+  def handle_action(option)
+    case option
+    when '1'
+      list_books
+    when '2'
+      list_people
+    when '3'
+      create_person
+    when '4'
+      create_book
+    when '5'
+      create_rental
+    when '6'
+      list_rentals
+    else
+      puts 'That is not a valid option'
+    end
+  end
+
+  def actions
+    puts
+    puts 'Please choose an option by entering a number:'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals for a given person id'
+    puts '7 - Exit'
+  end
+
+  def list_books
+    @books.each { |book| puts book }
+  end
```

Comment:
- [ ] Let's do some adjustments here so that book objects can be listed with some proper information. Right now, when we list all books, only the object memory references are displayed and not the book names. This way it's harder to know which book one wants to rent. See screenshot below:
  
  
  ![bookObjectRef](https://user-images.githubusercontent.com/45002654/147287330-a5501984-bb98-415d-9a48-6c3bc242573a.png)
  
  Consider working with the suggestion below:
  
  ```suggestion
    def list_books
      @books.each { |book| puts "Title: \"#{book.title}\", Author: #{book.author}" }
    end
  ```